build/root/Makefile

```diff
@@ -309,10 +309,7 @@ clean:
 else
 clean: clean_meta
 	build/make-clean.sh
-	rm -rf $(OUT_DIR)
-	rm -rf Godeps/_workspace # Just until we are sure it is gone
-	# TODO(thockin): Remove this when we call clean_generated.
-	rm -f pkg/generated/openapi/zz_generated.openapi.go
+	git clean -Xdf
```

Comment:
This seems a little aggressive to do without prompting the user.

And surprising.